background.js

```diff
@@ -145,6 +145,11 @@ function blockCookies(details) {
 	return {responseHeaders: responseHeaders};
 }
 
+// MS Edge support
+if (!chrome.webRequest) {
+  chrome.webRequest = browser.webRequest;
+}
```

Comment:
Commentary:

* Edge defines `window.chrome` already, but not `chrome.webRequest`.
* Edge defines `browser.webRequest` as callback-based. 
* Firefox defines `chrome.webRequest` as callback-based and `browser.webRequest` as Promise-based.
* Chrome defines `chrome.webRequest` as callback-based and does NOT define `browser.webRequest`.